.drone.yml

```
@@ -57,3 +57,114 @@ volumes:
   - name: docker
     host:
       path: /var/run/docker.sock
+
+---
+kind: pipeline
+name: arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+  - name: build
+    image: rancher/dapper:v0.6.0
+    commands:
+      - dapper ci
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: docker-publish-master
+    image: plugins/docker
+    settings:
+      build_args:
+        - ARCH=amd64
```

```suggestion
        - ARCH=arm64
```
Although it's not used in the dockerfile.